Exit with error status code when plan preview failed

TakumaKurosawa opened the pull request:
**What this PR does / why we need it**:

Plan preview CI send error comment but workflow is still succeeded.
So we have to exit with error status code when exception case.

**Which issue(s) this PR fixes**:

Fixes #4660

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
  - I'm expecting to plan-preview workflow failed when exception case.
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

khanhtc1202 commented:
@TakumaKurosawa Thanks for the great contributions to plan-preview codebase 😄 
There are some conflicts on this PR, could you update 🙏

TakumaKurosawa commented:
@khanhtc1202 
I'm not sure why this pull request is closed automatically.
I updated local master branch, then I rebased master branch into a head branch.

TakumaKurosawa commented:
@khanhtc1202 
Sorry.
Now I understand why😇
There is no changes on my branch, right? 😂
